docs/tutorials/multimodal/object_detection/quick_start/quick_start_coco.md

```diff
@@ -163,6 +170,73 @@ better_predictor.evaluate(test_path)
 For how to set those hyperparameters and finetune the model with higher performance, 
 see :ref:`sec_automm_detection_high_ft_coco`.
 
+### Inference
+Now that we have gone through the model setup, finetuning, and evaluation, this section details the inference. 
+Specifically, we layout the steps for using the model to make predictions and visualize the results.
+
+To run inference **on the entire test set**, perform:
+
+```python .input
+pred = predictor.predict(test_path)
+print(pred)
+```
+The output `pred` is a `pandas` `DataFrame` that has two columns, `image` and `bboxes`, where
+- in `image`, each row contains the image path
+- in `bboxes`, each row is a list of dictionaries, each one representing a bounding box: 
+  - `{"class": <predicted_class_name>, "bbox": [x1, y1, x2, y2], "score": <confidence_score>}`
+
+Note that, by default, the `predictor.predict` does not save the detection results into a file.
+
+To run inference and save results, run the following:
+```python .input
+pred = better_predictor.predict(test_path, save_results=True)
+```
+Here, we save `pred` into a `.txt` file, which exactly follows the same layout as in `pred`.
+You can use a predictor initialzed in anyway (i.e. finetuned predictor, predictor with pretrained model, etc.).
+Here, we demonstrate using the `better_predictor` loaded previously. 
+
+You can also download an image and **run inference on that single image**. The follow is an example:
+
+Download the example image:
+```python .input
+from autogluon.multimodal import download
+image_url = "https://raw.githubusercontent.com/dmlc/web-data/master/gluoncv/detection/street_small.jpg"
```

Comment:
Since this is a model pretrained on tiny_motorbike, does it make sense to use the street_small image as an example? Maybe we just pick one image from the test set of tiny_motorbike to do the inference and show results.

Reply:
Yea I agree. But I think it would be nice for the user to know how to inference on one single image, which is kinda different from inferencing on a dataset.